
  \documentclass[twoside]{article}
  \usepackage[affil-it]{authblk}
  \usepackage{lipsum} % Package to generate dummy text throughout this template
  \usepackage{eurosym}
  \usepackage[sc]{mathpazo} % Use the Palatino font
  \usepackage[T1]{fontenc} % Use 8-bit encoding that has 256 glyphs
  \usepackage[utf8]{inputenc}
  \linespread{1.05} % Line spacing-Palatino needs more space between lines
  \usepackage{microtype} % Slightly tweak font spacing for aesthetics\[IndentingNewLine]
  \usepackage[hmarginratio=1:1,top=32mm,columnsep=20pt]{geometry} % Document margins
  \usepackage{multicol} % Used for the two-column layout of the document
  \usepackage[hang,small,labelfont=bf,up,textfont=it,up]{caption} % Custom captions under//above floats in tables or figures
  \usepackage{booktabs} % Horizontal rules in tables
  \usepackage{float} % Required for tables and figures in the multi-column environment-they need to be placed in specific locations with the[H] (e.g. \begin{table}[H])
  \usepackage{hyperref} % For hyperlinks in the PDF
  \usepackage{lettrine} % The lettrine is the first enlarged letter at the beginning of the text
  \usepackage{paralist} % Used for the compactitem environment which makes bullet points with less space between them
  \usepackage{abstract} % Allows abstract customization
  \renewcommand{\abstractnamefont}{\normalfont\bfseries} 
  %\renewcommand{\abstracttextfont}{\normalfont\small\itshape} % Set the abstract itself to small italic text\[IndentingNewLine]
  \usepackage{titlesec} % Allows customization of titles
  \renewcommand\thesection{\Roman{section}} % Roman numerals for the sections
  \renewcommand\thesubsection{\Roman{subsection}} % Roman numerals for subsections
  \titleformat{\section}[block]{\large\scshape\centering}{\thesection.}{1em}{} % Change the look of the section titles
  \titleformat{\subsection}[block]{\large}{\thesubsection.}{1em}{} % Change the look of the section titles
  \usepackage{fancyhdr} % Headers and footers
  \pagestyle{fancy} % All pages have headers and footers
  \fancyhead{} % Blank out the default header
  \fancyfoot{} % Blank out the default footer
  \fancyhead[C]{X-meeting $\bullet$ October 2019 $\bullet$ Campos do  Jord\~ao} % Custom header text
  \fancyfoot[RO,LE]{} % Custom footer text
  %----------------------------------------------------------------------------------------
  % TITLE SECTION
  %---------------------------------------------------------------------------------------- 
 
 \title{\vspace{-15mm}\fontsize{24pt}{10pt}\selectfont\textbf{ Machado: a genomic data integration framework for Chado developed with Django }} % Article title
  
  
  \author{ MAURICIO DE ALVARENGA MUDADU, Adhemar Zerlotini Neto }
  
  \affil{ Embrapa Inform\'atica Agropecu\'aria }
  \vspace{-5mm}
  \date{}
  
  %---------------------------------------------------------------------------------------- 
  
  \begin{document}
  
  
  \maketitle % Insert title
  
  
  \thispagestyle{fancy} % All pages have headers and footers
  %----------------------------------------------------------------------------------------  
  % ABSTRACT
  
  %----------------------------------------------------------------------------------------  
  
  \begin{abstract}
  Technological advances in biological research has led to a data deluge which have great impact in agriculture,  especially in plant and animal breeding. In this regard,  genome projects and multiomics experiments generate huge volumes of data that must be stored,  mined and transformed into useful knowledge. Furthermore,  all this information is supposed to be accessible and,  if possible,  browseable afterwards. Computational biologists have been dealing with this scenario for over a decade and have been implementing software libraries,  toolkits,  platforms,  and databases to succeed in this matter.
Although public wide databases exist,  research groups still struggle to store and analyze data with local resources and expertise. The GMOD,  or Generic Model Organism Database project,  is currently the initiative that made the most advance in producing a "collection of open source software tools for managing,  visualizing,  storing and disseminating genetic and genomic data". Its biological relational database schema known as Chado is widely adopted and many softwares are able to connect to it. Such softwares usually contain a set of scripts to preprocess the data files in order to provide visualization and search capabilities,  but in fact they can only operate independently. 
The Embrapaâ€™s Bioinformatics Multi-user Laboratory have been developing an open source software known as machado,  that has a Django model to connect to Chado,  thus avoiding extra efforts to make data compatible to the database schema. The machado software has several data loading tools for genomic and transcriptomic data and also for annotation results for tools such as BLAST,  InterproScan,  OrthoMCL and lsTrap. Machado has also an API to connect to Jbrowse which can be easily setup. A web browsing visualisation tool is implemented using the Apache2 server alongside with Django WSGI library. The haystack software integrated with the elasticsearch engine was used to create an index for querying the data using keywords,  identification and annotation entries. Caching is also enabled for fast data retrieving.
This project aims to contribute to the research community by producing a modern object-relational framework to store,  integrate,  query,  and visualize all the major genomics data types. Such endeavour takes advantage of the latest Python modules to produce an effective open source resource and facilitate the identification of specific biological components related to economic traits in agriculture. Machado is available at GitHub: https://github.com/lmb-embrapa/machado .
  
  Funding: Embrapa \\ 
  \end{abstract}
  \end{document} 