
  \documentclass[twoside]{article}
  \usepackage[affil-it]{authblk}
  \usepackage{lipsum} % Package to generate dummy text throughout this template
  \usepackage{eurosym}
  \usepackage[sc]{mathpazo} % Use the Palatino font
  \usepackage[T1]{fontenc} % Use 8-bit encoding that has 256 glyphs
  \usepackage[utf8]{inputenc}
  \linespread{1.05} % Line spacing-Palatino needs more space between lines
  \usepackage{microtype} % Slightly tweak font spacing for aesthetics\[IndentingNewLine]
  \usepackage[hmarginratio=1:1,top=32mm,columnsep=20pt]{geometry} % Document margins
  \usepackage{multicol} % Used for the two-column layout of the document
  \usepackage[hang,small,labelfont=bf,up,textfont=it,up]{caption} % Custom captions under//above floats in tables or figures
  \usepackage{booktabs} % Horizontal rules in tables
  \usepackage{float} % Required for tables and figures in the multi-column environment-they need to be placed in specific locations with the[H] (e.g. \begin{table}[H])
  \usepackage{hyperref} % For hyperlinks in the PDF
  \usepackage{lettrine} % The lettrine is the first enlarged letter at the beginning of the text
  \usepackage{paralist} % Used for the compactitem environment which makes bullet points with less space between them
  \usepackage{abstract} % Allows abstract customization
  \renewcommand{\abstractnamefont}{\normalfont\bfseries} 
  %\renewcommand{\abstracttextfont}{\normalfont\small\itshape} % Set the abstract itself to small italic text\[IndentingNewLine]
  \usepackage{titlesec} % Allows customization of titles
  \renewcommand\thesection{\Roman{section}} % Roman numerals for the sections
  \renewcommand\thesubsection{\Roman{subsection}} % Roman numerals for subsections
  \titleformat{\section}[block]{\large\scshape\centering}{\thesection.}{1em}{} % Change the look of the section titles
  \titleformat{\subsection}[block]{\large}{\thesubsection.}{1em}{} % Change the look of the section titles
  \usepackage{fancyhdr} % Headers and footers
  \pagestyle{fancy} % All pages have headers and footers
  \fancyhead{} % Blank out the default header
  \fancyfoot{} % Blank out the default footer
  \fancyhead[C]{X-meeting $\bullet$ October 2019 $\bullet$ Campos do  Jord\~ao} % Custom header text
  \fancyfoot[RO,LE]{} % Custom footer text
  %----------------------------------------------------------------------------------------
  % TITLE SECTION
  %---------------------------------------------------------------------------------------- 
 
 \title{\vspace{-15mm}\fontsize{24pt}{10pt}\selectfont\textbf{ Fcoex: an R package for detecting co-expression modules in single-cell RNA-Seq data }} % Article title
  
  
  \author{ Tiago Lubiana, HELDER T I NAKAYA }
  
  \affil{ Institute of Mathematics and Statistics,  University of S\~ao Paulo }
  \vspace{-5mm}
  \date{}
  
  %---------------------------------------------------------------------------------------- 
  
  \begin{document}
  
  
  \maketitle % Insert title
  
  
  \thispagestyle{fancy} % All pages have headers and footers
  %----------------------------------------------------------------------------------------  
  % ABSTRACT
  
  %----------------------------------------------------------------------------------------  
  
  \begin{abstract}
  The boom of single-cell transcriptomics was followed by a growth in methods for the analysis of single-cell data. Currently,  standard pipelines (such as Seurat and Biocondutorâ€™s OSCA) do not include co-expression network building and modules detection methods. Modern systems biology uses co-expression networks both for exploratory data analysis and gene regulatory network inference. Current methods for building these networks,  such as WGCNA,  were developed for bulk RNA-Seq and do not perform as well in single-cell data. In the present work,  we show how a feature selection algorithm,  the Fast Correlation-Based Filter (FCBF),  can be used to detect co-expression modules in single-cell data via an R package called fcoex. The package is awaiting reviews for the Bioconductor repository and is available at https://github.com/csbl-usp/fcoex. We applied it to single-cell data from human,  mice,  and zebrafish,  detecting o co-expression modules with known biological partners and putative associations. The presence of anticorrelated genes in the same modules allowed the detection,  in the zebrafish dataset,  of a module containing both a ligand (apela) and its receptors (aplnra/aplnrb),  yielding insights into the biology of vertebrate development. Also,  fcoex enables module-based reclustering of the datasets for multilevel labeling of cells,  uncovering new populations,  and avoiding trade-offs of the current label-determination methods. In parallel,  we detected new candidates for subpopulations of zebrafish embryo cells and human blood monocytes,  demonstrating the usefulness of our tools for exploratory data analysis of single cells.
  
  Funding: This work was supported by the grant 2018/10257-2,  S\~ao Paulo Research Foundation (FAPESP) \\ 
  \end{abstract}
  \end{document} 