
  \documentclass[twoside]{article}
  \usepackage[affil-it]{authblk}
  \usepackage{lipsum} % Package to generate dummy text throughout this template
  \usepackage{eurosym}
  \usepackage[sc]{mathpazo} % Use the Palatino font
  \usepackage[T1]{fontenc} % Use 8-bit encoding that has 256 glyphs
  \usepackage[utf8]{inputenc}
  \linespread{1.05} % Line spacing-Palatino needs more space between lines
  \usepackage{microtype} % Slightly tweak font spacing for aesthetics\[IndentingNewLine]
  \usepackage[hmarginratio=1:1,top=32mm,columnsep=20pt]{geometry} % Document margins
  \usepackage{multicol} % Used for the two-column layout of the document
  \usepackage[hang,small,labelfont=bf,up,textfont=it,up]{caption} % Custom captions under//above floats in tables or figures
  \usepackage{booktabs} % Horizontal rules in tables
  \usepackage{float} % Required for tables and figures in the multi-column environment-they need to be placed in specific locations with the[H] (e.g. \begin{table}[H])
  \usepackage{hyperref} % For hyperlinks in the PDF
  \usepackage{lettrine} % The lettrine is the first enlarged letter at the beginning of the text
  \usepackage{paralist} % Used for the compactitem environment which makes bullet points with less space between them
  \usepackage{abstract} % Allows abstract customization
  \renewcommand{\abstractnamefont}{\normalfont\bfseries} 
  %\renewcommand{\abstracttextfont}{\normalfont\small\itshape} % Set the abstract itself to small italic text\[IndentingNewLine]
  \usepackage{titlesec} % Allows customization of titles
  \renewcommand\thesection{\Roman{section}} % Roman numerals for the sections
  \renewcommand\thesubsection{\Roman{subsection}} % Roman numerals for subsections
  \titleformat{\section}[block]{\large\scshape\centering}{\thesection.}{1em}{} % Change the look of the section titles
  \titleformat{\subsection}[block]{\large}{\thesubsection.}{1em}{} % Change the look of the section titles
  \usepackage{fancyhdr} % Headers and footers
  \pagestyle{fancy} % All pages have headers and footers
  \fancyhead{} % Blank out the default header
  \fancyfoot{} % Blank out the default footer
  \fancyhead[C]{X-meeting eXperience $\bullet$ November 2020} % Custom header text
  \fancyfoot[RO,LE]{} % Custom footer text
  %----------------------------------------------------------------------------------------
  % TITLE SECTION
  %---------------------------------------------------------------------------------------- 
 
 \title{\vspace{-15mm}\fontsize{24pt}{10pt}\selectfont\textbf{ Complexity Analysis of Algorithms: A case study about Bioinformatics Tools }} % Article title
  
  
  \author{ Luan Ribeiro Siqueira,  Vict\'oria Cardoso dos Santos,  M\^onica Silva de Oliveira,  Adonney Allan de Oliveira Veras,  Gislenne da Silva Moia }
  
  \affil{ UNIVERSIDADE FEDERAL DO PAR\'A }
  \vspace{-5mm}
  \date{}
  
  %---------------------------------------------------------------------------------------- 
  
  \begin{document}
  
  
  \maketitle % Insert title
  
  
  \thispagestyle{fancy} % All pages have headers and footers
  %----------------------------------------------------------------------------------------  
  % ABSTRACT
  
  %----------------------------------------------------------------------------------------  
  
  \begin{abstract}
  The diverse analysis performed by omics sciences,  driven by the reduction of costs in the DNA sequencing and reduction of the total time to carry out this process,  resulted in an exponential increase in the deposit of all this information in public databases,  for example,  the National Center for Biotechnology Information - NCBI. The volume of data produced by these sequencing platforms demanded the development of algorithms capable of performing the most varied analysis,  such as remotion of redundancy in raw reads from the sequencing process. However,  it is worth mentioning that the existence of these various tools which performs this task with proven accuracy through their scientific publications,  they do not analyze criteria related to the algorithmic complexity involved in their development. Therefore,  this work demonstrated an analysis of algorithmic complexity,  through empirical analysis already described in the literature,  this analysis was performed with sixteen raw reads datasets with sizes ranging from 900 thousand to 12 million,  they were obtained in the NCBI database in the Sequence Read Archive (SRA) format,  they were converted to the FASTQ standard through the fastq-dump tool,  the selected tools were: MarDRe,  NGSReadsTreatment,  ParDRe,  FastUniq,  and BioSeqZip. The analysis was performed on the R statistic platform,  by using the GuessCompx package using the processing time of all datasets required by each tool as input,  the models created were submitted to the glm adjustment function,  in order to identify the function that indicates the complexity observed in each model. To this end,  seven Big-O notations were observed:O(n),  O(log(n)),  O(n$^2$),  O(n$^3$),  O(1),  O(n-log-n) and O(2$^n$). With the analysis of the results plotted graphically,  it can be concluded that the NGSReadsTreatment tool obtained the least complexity in the processing of the datasets used in this analysis,  presenting a linear complexity behavior,  which leads us to infer that for datasets with high volume,  this tool shows an interesting alternative to performing data processing.
  
  Funding:   \\
  \href{http://ab3c.org.br/xpress_pres2020/xmxp2020-297674.html}{Link to Video:}

  \end{abstract}
   
  \end{document} 