
  \documentclass[twoside]{article}
  \usepackage[affil-it]{authblk}
  \usepackage{lipsum} % Package to generate dummy text throughout this template
  \usepackage{eurosym}
  \usepackage[sc]{mathpazo} % Use the Palatino font
  \usepackage[T1]{fontenc} % Use 8-bit encoding that has 256 glyphs
  \usepackage[utf8]{inputenc}
  \linespread{1.05} % Line spacing-Palatino needs more space between lines
  \usepackage{microtype} % Slightly tweak font spacing for aesthetics\[IndentingNewLine]
  \usepackage[hmarginratio=1:1,top=32mm,columnsep=20pt]{geometry} % Document margins
  \usepackage{multicol} % Used for the two-column layout of the document
  \usepackage[hang,small,labelfont=bf,up,textfont=it,up]{caption} % Custom captions under//above floats in tables or figures
  \usepackage{booktabs} % Horizontal rules in tables
  \usepackage{float} % Required for tables and figures in the multi-column environment-they need to be placed in specific locations with the[H] (e.g. \begin{table}[H])
  \usepackage{hyperref} % For hyperlinks in the PDF
  \usepackage{lettrine} % The lettrine is the first enlarged letter at the beginning of the text
  \usepackage{paralist} % Used for the compactitem environment which makes bullet points with less space between them
  \usepackage{abstract} % Allows abstract customization
  \renewcommand{\abstractnamefont}{\normalfont\bfseries} 
  %\renewcommand{\abstracttextfont}{\normalfont\small\itshape} % Set the abstract itself to small italic text\[IndentingNewLine]
  \usepackage{titlesec} % Allows customization of titles
  \renewcommand\thesection{\Roman{section}} % Roman numerals for the sections
  \renewcommand\thesubsection{\Roman{subsection}} % Roman numerals for subsections
  \titleformat{\section}[block]{\large\scshape\centering}{\thesection.}{1em}{} % Change the look of the section titles
  \titleformat{\subsection}[block]{\large}{\thesubsection.}{1em}{} % Change the look of the section titles
  \usepackage{fancyhdr} % Headers and footers
  \pagestyle{fancy} % All pages have headers and footers
  \fancyhead{} % Blank out the default header
  \fancyfoot{} % Blank out the default footer
  \fancyhead[C]{X-meeting eXperience $\bullet$ November 2020} % Custom header text
  \fancyfoot[RO,LE]{} % Custom footer text
  %----------------------------------------------------------------------------------------
  % TITLE SECTION
  %---------------------------------------------------------------------------------------- 
 
 \title{\vspace{-15mm}\fontsize{24pt}{10pt}\selectfont\textbf{ A database of viral taxon-specific profile HMMs for the detection and classification of viral sequences }} % Article title
  
  
  \author{ Wendel Hime Lima Castro,  Arthur Gruber,  Liliane Santana Oliveira Kashiwabara }
  
  \affil{ UNIVERSIDADE TECNOL\'OGICA FEDERAL DO PARAN\'A,  UNIVERSIDADE DE S\~AO PAULO }
  \vspace{-5mm}
  \date{}
  
  %---------------------------------------------------------------------------------------- 
  
  \begin{document}
  
  
  \maketitle % Insert title
  
  
  \thispagestyle{fancy} % All pages have headers and footers
  %----------------------------------------------------------------------------------------  
  % ABSTRACT
  
  %----------------------------------------------------------------------------------------  
  
  \begin{abstract}
  Viruses are the most abundant and diverse biological entities on nature,  characterized by high rates of replication and evolution. The highly variable nature of their genomes,  composed of either single- or double-stranded RNA or DNA molecules,  implies that different enzymes are required for replication. No single gene is shared by all viruses and,  therefore,  there are no universal markers for viral phylogenetic analysis and taxonomic classification. Thus,  different viral markers must be established to study distinct taxonomic groups. Profile HMMs are statistical models that incorporate the diversity of a set of primary sequences and constitute a very sensitive approach for detecting known and emerging viruses. Publicly available resources of viral profile HMMs are based on orthologous clusters,  which may include sequences from different taxonomic groups,  implying that models can often detect a wide and unpredictable range of taxa. We have recently developed TABAJARA,  a program for the rational design of profile HMMs. TABAJARA uses a multiple sequence alignment (MSA) as input and is able to find short blocks that are either (1) conserved across all sequences or (2) discriminative for two specific groups of sequences. The program then runs validation tests against the training set and automatically define cutoff scores customized for each developed model. Alternatively,  full-length sequences are used for model construction and highly specific profile HMMs may be obtained when using the assigned cut-off scores. This approach has been successfully used to develop models for some taxonomically specific viral groups,  including Microviridae phages and viruses of the genus Flavivirus. Given the utmost importance of developing bioinformatic resources and tools for novel virus detection and classification,  we decided to extend our approach to all viral taxa represented with protein sequences on the NCBIâ€™s Identical Protein Groups (IPG) database. We implemented a Python pipeline to perform the following steps: (1) automatic sequence retrieval from IPG database,  according to a pre-defined list of queries; (2) data organization and storage; (3) multiple sequence alignment for each query; (4) profile HMM construction; (5) model validation; and (6) report generation. We obtained 20.749 models,  comprising prokaryotic and eukaryotic viruses. Relational searches can be performed on a web front end according to the taxonomic name or ID,  protein name,  type of model (short or full-length) and host (prokaryotic or eukaryotic),  among other parameters. The selected models can then be downloaded. We expect this database will become an important resource for the scientific community.
  
  Funding:   \\
  \href{http://ab3c.org.br/xpress_pres2020/xmxp2020-294413.html}{Link to Video:}

  \end{abstract}
   
  \end{document} 